data/templates/get_driveid.js

```diff
@@ -185,6 +192,18 @@ function buildDriveMap(wwidData, vdData, scsiData) {
                 }
             }
         });
+        //vmhbaAdapter:CChannel:TTarget:LLUN
+        //Adapter is set to default value 32 for only one eUsb
+        if (esxiWwid[1] === "mpx.") {
+            esxiWwid[1] = '';
+            if (scsiId !== '') {
+                var hctl = scsiId.split(":");
+                if (hctl.length === 4) {
+                    esxiWwid[1] = "mpx.vmhba32:C".concat(hctl[1], ":T", hctl[2], ":L", hctl[3]);
```

Comment:
32 is the USB controller id. Servers use USB controller ID integrated in PCH. It is OK to only include one id here.